src/index.ts

```diff
@@ -159,11 +164,43 @@ export class MondayRedisService extends BaseConnector {
               this.serialize(item[title], title),
             )
           ),
-          this.redis.hset(this.namesKey, item.name, item.id)
+          this.redis.hset(this.namesKey, item.name, item.id),
         ]))
-        .flat()
+          .flat()
       )
     }
+
+    this.boardId = await this.getBoardId()
+    this.monday.on(
+      {
+        type: 'ChangeColumnValue',
+        boardId: this.boardId,
+      },
+      async (event) => {
+        const { boardId, itemId, columnTitle, value: { value } } = event
+        if (this.boardId === parseInt(boardId, 10)) {
+          const redisCurrentValue = await this.redis.hget(
+            this.keyForItem(itemId),
+            columnTitle,
+          )
+          const redisCurrentValueDeserialized = this.deserialize(redisCurrentValue, columnTitle)
+
+          if (redisCurrentValueDeserialized !== value) {
+            this.app.getLogger().info(`Monday event received - Redis/Monday values differ for itemId ${itemId} (title: ${columnTitle})`)
+            await this.redis.hset(
+              this.keyForItem(itemId),
+              columnTitle,
+              this.serialize(value, columnTitle),
+            )
+            const currentChanges = await this.redis.get(this.changesKey)
```

Comment:
You can't do this. This is not atomic and unsafe. This is why I suggested calling "destageChages" with the change entry to ignore. It is designed to be atomic.